collections/editor/occurrencetabledisplay.php

```diff
@@ -154,7 +154,7 @@ function () {
 </head>
 <body style="margin-left: 0px; margin-right: 0px;background-color:white;">
 	<a class="skip-link" href="#skip-search"><?php echo $LANG['SKIP_SEARCH'] ?></a>
-	<div id="innertext">
+	<div id="innertext" style="max-width: 100vw">
```

Comment:
I'm noting for posterity that this inline styling was necessary in order to override styling for the #innertext class more broadly.